Lib-O5e-MonsterSupplement/o5e_ExtDB_merge.command

```diff
@@ -0,0 +1,19 @@
+[h: o5eResponse = arg (0)]
+[h: url = arg (1)]
+[h: urlTokens = stringToList (url, "=")]
+[h, if (listCount (urlTokens) > 1): slug = decode (listGet (urlTokens, 1)); assert (0, url + " is an invalid search token")]
+[h: o5e_ExtDB_Constants (getMacroName())]
+[h: log.debug (CATEGORY + "slug = " + slug)]
+
+[h: responseCount = json.get (o5eResponse, "count")]
+[h, if (responseCount == ""): responseCount = 0]
+[h: responseResults = json.get (o5eResponse, "results")]
+[h: extDb = o5e_ExtDB_getDB()]
+[h: extDbResults = json.path.read (extDb, ".[?(@.monsterDBName =~ /.*" + slug + ".*/i)]")]
+[h: log.debug (CATEGORY + "extDbResults = " + extDbResults)]
+[h: extDbResultsCount = json.length (extDbResults)]
+[h: responseCount = responseCount + extDbResultsCount]
+[h: responseResults = json.merge (responseResults, extDbResults)]
```

Comment:
Before merging extDbResults, make sure it's not empty; Doing all this work with an empty JSON object is stupid. Also, after merging, re-sort the responseResults fields. Otherwise the user gets a list of choices where all the custom created content is at the bottom.